includes/discovery/route/routeros.inc.php

```diff
@@ -19,56 +19,65 @@
  * @package    LibreNMS
  * @link       https://www.librenms.org
  *
- * @copyright  2022 Peca Nesovanovic
+ * @copyright  2025 Peca Nesovanovic
  *
  * @author     Peca Nesovanovic <peca.nesovanovic@sattrakt.com>
  */
 
+use App\Facades\PortCache;
+use Illuminate\Support\Facades\Log;
+use LibreNMS\Exceptions\InvalidIpException;
 use LibreNMS\Util\IPv6;
 
-$oids = SnmpQuery::walk('IPV6-MIB::ipv6RouteTable')->table(1);
+$oids = SnmpQuery::cache()->hideMib()->walk('IPV6-MIB::ipv6RouteTable')->table(3);
```

Comment:
Do not cache a walk that is only walked once, it wastes memory.

Reply:
done